### PR TITLE
[MIG]10.0-purchase operating unit

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
===============================
Purchase with Operating Units
===============================

This module introduces the following features:
- It introduces the operating unit to the purchase order.
- The operating unit is copied to the invoice.
- The operating unit is copied to the stock picking.
- It implements user's security rules.

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]

File: purchase_operating_unit/views/purchase_order_view.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                <field name="operating_unit_id" groups="operating_unit.group_multi_operating_unit"/>
+<!--                          on_change="onchange_operating_unit_id(requesting_operating_unit_id,operating_unit_id)"/> -->
+            </field>
+            <xpath expr="//field[@name='order_line']"
+                 position="attributes">
+                <attribute name="context">{'operating_unit_id': operating_unit_id}</attribute>
+            </xpath>
+<!--                 <field name="warehouse_id" position="attributes"> -->
+<!--                     <attribute name="on_change">onchange_warehouse_id(warehouse_id,requesting_operating_unit_id,operating_unit_id)</attribute> -->
+<!--                 </field> -->
```
Comment (LoisRForgeFlow) [inline purchase_operating_unit/views/purchase_order_view.xml]: remove commented code.

File: purchase_operating_unit/views/purchase_order_view.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase_order_form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <field name="partner_ref" position="after">
+                <field name="requesting_operating_unit_id" groups="operating_unit.group_multi_operating_unit"/>
+<!--                          on_change="onchange_operating_unit_id(requesting_operating_unit_id,operating_unit_id)"/> -->
+                <field name="operating_unit_id" groups="operating_unit.group_multi_operating_unit"/>
+<!--                          on_change="onchange_operating_unit_id(requesting_operating_unit_id,operating_unit_id)"/> -->
```
Comment (LoisRForgeFlow) [inline purchase_operating_unit/views/purchase_order_view.xml]: remove commented code.

File: purchase_operating_unit/views/purchase_order_view.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    </record>
+
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase_order_form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <field name="partner_ref" position="after">
+                <field name="requesting_operating_unit_id" groups="operating_unit.group_multi_operating_unit"/>
+<!--                          on_change="onchange_operating_unit_id(requesting_operating_unit_id,operating_unit_id)"/> -->
```
Comment (LoisRForgeFlow) [inline purchase_operating_unit/views/purchase_order_view.xml]: remove commented code.

**Review by LoisRForgeFlow — APPROVED** [review]
Minor comments, otherwise LGTM.

**Pull request merged**